node/_crypto/scrypt_test.ts

```diff
@@ -0,0 +1,179 @@
+import { scrypt, scryptSync } from "./scrypt.ts";
```

Comment:
nit: This file needs copyright header

```
// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
```

Reply:
Let's address this in another PR as we already have many files missing copyright headers.

Reply:
ok no problem

and thanks for merging!

deno_std/node/crypto is getting closer to node.js :D